boards/arm/mec172xmodular_assy6930/support/spi_cfg_4MBit.txt

```diff
@@ -0,0 +1,54 @@
+; MEC1727 SPI Image Generator configuration file for internal 512KB flash
+[SPI]
+SPISizeMegabits = 4
+Flashmap = false
+FlshmapAddr = 0
+
+[DEVICE]
+TagAddr0 = 0
+TagAddr1 = 0
+; BoardID is used by a Boot-ROM feature named PlatformID. By default PlatformID
+; is disabled. If PlatformID is enabled in OTP Boot-ROM will check the image
+; BoardID value with an OTP PlatformID value. Load is aborted if the ID's do
+; not match.
+BoardID = 0
+
+[IMAGE "0"]
+ImageLocation = 0x1000
```

Comment:
@wu-frances consider using 0x100 which is most common location when using this board within a host in integrated setup (MAF).